src/protocol/modulus_conversion/convert_shares.rs

```diff
@@ -108,6 +108,36 @@ impl ConvertShares {
     }
 }
 
+pub async fn convert_all_shares<F: Field>(
+    ctx: &ProtocolContext<'_, F>,
+    input: &[u64],
+) -> Result<Vec<Vec<Replicated<F>>>, BoxError> {
+    let converted_shares = try_join_all(zip(repeat(ctx), input).enumerate().map(
+        |(record_id, (ctx, row))| async move {
+            ConvertShares::new(XorShares {
+                num_bits: 64,
```

Comment:
Let's make this a param - not a hardcoded value. 

For example, if we are only using 40-bit match keys, then we only need to convert 40 bits of this thing.

Reply:
Done!

Reply:
What do you mean? This is still not a parameter to the function. It's still hard-coded.